sys/include/net/ng_netconf.h

```diff
@@ -119,8 +119,12 @@ typedef enum {
      * @note not all transceivers may support this interrupt
      */
     NETCONF_OPT_TX_END_IRQ,
-    NETCONF_OPT_AUTOCCA,            /**< en/disable to check automatically
-                                         before sending the channel is clear. */
+    NETCONF_OPT_CSMA,               /**< en/disable to check automatically
```

Comment:
I don't have the 2006 version, but to my understanding that's exactly what I thought: sensing the channel is backed off, not the sending.